src/main/java/org/spongepowered/api/entity/living/Living.java

```diff
@@ -294,23 +294,4 @@
      * @param invisible Whether this entity is invisible or not
      */
     void setInvisible(boolean invisible);
-
-    /**
-     * Gets whether this living entity is invisible to the specific
-     * {@link Player} entity.
-     *
-     * @param player The other player to check
-     * @return Whether this entity is invisible to the given player
-     */
-    boolean isInvisibleTo(Player player);
```

Comment:
[You had said](https://github.com/SpongePowered/SpongeAPI/pull/440#discussion_r25787602) that it had already been added.